development/editor/introduction.rst

```diff
@@ -0,0 +1,9 @@
+.. _doc_editor_intro:
+
+Introduction
+============
+
+As the engine is constantly evolving and new features are being added, it is
+important to familiarize yourself with the editor and tools development in C++.
+Without the editor, some of the engine's internal functionality would become
+unaccessible, make the development more cumbersome or not intuitive.
```

Comment:
As an editor-based engine, it's a bit tautological to say that it would be more cumbersome without editor though :P

Reply:
If you think about it "as-if", it makes it more logical. :)

Wonder whether the whole sentence could be dropped, or just remove `cumbersome`.